components/org.wso2.micro.integrator.security/src/main/java/org/wso2/micro/integrator/security/user/core/internal/Activator.java

```diff
@@ -36,34 +34,36 @@
  * Therefore we read properties but do not keep a reference to it.
  */
 public class Activator extends BundleCheckActivator {
+
     private static final Log log = LogFactory.getLog(Activator.class);
 
     public void startDeploy(BundleContext bundleContext) throws Exception {
+
+        if (Boolean.parseBoolean(System.getProperty("NonUserCoreMode"))) {
+            return;
+        }
+
         // Need permissions in order to instantiate user core
         SecurityManager secMan = System.getSecurityManager();
 
-		/*
+        /*
          * Read the SSL trust store configurations from the Security.TrustStore
-		 * element of the Carbon.xml
-		 */
-        CarbonServerConfigurationService config = CarbonServerConfigurationService.getInstance();
-        String type = config.getFirstProperty("Security.TrustStore.Type");
-        String password = config.getFirstProperty("Security.TrustStore.Password");
-        String storeFile =
-                new File(config.getFirstProperty("Security.TrustStore.Location")).getAbsolutePath();
-        // set the SSL trust store System Properties
-        System.setProperty("javax.net.ssl.trustStore", storeFile);
-        System.setProperty("javax.net.ssl.trustStoreType", type);
-        System.setProperty("javax.net.ssl.trustStorePassword", password);
+         * element of the Carbon.xml
+         */
+//        CarbonServerConfigurationService config = CarbonServerConfigurationService.getInstance();
```

Comment:
Shall we remove the commented out parts?

Reply:
+1 Fixed